src/software/world/world.cpp

```diff
@@ -39,7 +39,7 @@ void World::updateFieldGeometry(const Field &new_field_data)
     updateTimestamp(getMostRecentTimestampFromMembers());
 }
 
-void World::updateBallState(const BallState &new_ball_state)
+void World::updateBallStateWithTimestamp(const TimestampedBallState &new_ball_state)
```

Comment:
```suggestion
void World::updateTimestampedBallState(const TimestampedBallState &new_ball_state)
```
should this be renamed for consistency?

Reply:
Good suggestion. I'm sort of inclined to keep it as is for now because it's a bit more descriptive. ie. it's more clear we are updating the ball with a new state and some correlated timestamp. Technically the datatype being used for this is irrelevant.

However we've got plans to revisit the world datastructures in the future and will likely come back to this. Need to make a ticket still but will include this in the things to investigate.